DEVELOPING.md

```diff
@@ -17,7 +32,131 @@ This project:
 
 3. uses [Swift Package Manager](https://github.com/apple/swift-package-manager) to manage dependencies for development. Cocapods is supported for *consuming* the SDK.
 
-4. uses [secret-shield](https://github.com/mapbox/secret-shield) to help block secrets such as access tokens from being exposed. Install `secret-shield` by entering `npm install -g @mapbox/secret-shield`. Install the pre-commit hook by running `scripts/install-pre-commit/install-pre-commit.sh`
+4. uses [secret-shield](https://github.com/mapbox/secret-shield) to help block secrets such as access tokens from being exposed. Setup `secret-shield` by running:
+```
+npm install -g @mapbox/secret-shield
+scripts/install-pre-commit/install-pre-commit.sh
+```
+
+5. uses CircleCI and AWS Device Farm for continuous integration.
```

Comment:
Should be updated to refer to Firebase Test Lab now (check for other references as well)